ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -77,6 +110,49 @@ objects:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
     containers:
+    - name: egress-proxy
+      command:
+      - /bin/bash
+      - -c
+      args:
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        if [[ "${PROXY_ENABLED:-false}" == "true" ]]; then
```

Comment:
nit: you don't actually need the `:-false` here; when the variable is unset it will just compare the empty string with `true` and not match.  For precedent, see [here][1].

[1]: https://github.com/openshift/release/blob/7c64c435e59d47ab7457c53a3c00454bca6d7fd4/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L192

Reply:
oops, i thought i removed those all when i was thinking I needed to add defaults in... ack i'll remove